### DISCUSSION TITLE
Nuovo episodio domani

### DISCUSSION THREAD

**Discussion by pdonadeo** [pdonadeo/e-uscito-joypad#29] [General]
Casomai ve lo foste perso su Instagram...

https://instagram.com/stories/corrisaltaspara/2864918312569228256?igshid=MDJmNzVkMjY=

**Comment by MarcoJul** [comment]
Siamo liberi di usare PUNTATA in giro per tutto il sito quindi!😆

**Reply by pdonadeo** [reply]
A questo punto devo ammettere che la mia mente semplice è confusa.

Non so come interpretare le sconcertanti dichiarazioni di Zampa: come una presa d'atto della caducità della vita, nella quale la diatriba "episodio vs. puntata" scompare di fronte ad argomenti come "la vita, l'universo e tutto quanto"… oppure siamo di fronte ad una resa senza condizioni di fronte ad un avversario tirannico, in questo caso Bordone?

<br /><br /><br /><br /><br />

_P.S. Forza, fate uscire una bella discussione stupida ed inutile, per spezzare un po' le questioni tecniche che mal si conciliano con questo caldo mortale!_

**Reply by agofa** [reply]
Vi credo sulla fiducia, non ho ancora sentito la punt..l'episod...l'edizion...la version...

Però so già che il mio psicanalista mi dirà di stare alla larga da queste discussioni.
E comunque nessuno e nulla può contro Bordone

**Reply by agofa** [reply]
Ah dimenticavo!
Non mi fregate!

https://www.euscitojoypad.it/ dice che non è uscita la puntepisedizvers quindi non è uscita
Ecco.

**Reply by MarcoJul** [reply]
Perché la piattaforma viola non fa testo, io non la registro mentalmente come puntata ascoltata senza i suonini di intermezzo fra un blocco e l'altro.

Comunque si parla tanto male dei polacchi e delle loro promesse mal mantenute e poi crolla così facilmente un mito etimologico come la conservazione della parola "episodio".. non ho più certezze, stasera butto giù tutto il frontend e lo riscrivo in Jquery💀

Piccolo spoiler innocuo (anche se la dannosità é nelle orecchie di chi la riceve): auguri a chi dovrà sbobinare quest'ultimo episodio😆

**Reply by agofa** [reply]
> non ho più certezze, stasera butto giù tutto il frontend e lo riscrivo in Jquery

Con tutte le vocali cambiate in "i" però
